src/EVault/shared/types/LTVConfig.sol

```diff
@@ -29,43 +31,51 @@ library LTVConfigLib {
         return self.targetTimestamp != 0;
     }
 
-    // Get current LTV of a collateral. When liquidation LTV is lowered, it is ramped down to target value over a period of time.
+    // Get current LTV of the collateral. When liquidation LTV is lowered, it is ramped down to the target value over a period of time.
     function getLTV(LTVConfig memory self, bool liquidation) internal view returns (ConfigAmount) {
-        if (!liquidation || block.timestamp >= self.targetTimestamp || self.targetLTV >= self.originalLTV) {
-            return self.targetLTV;
+        if (!liquidation) {
+            return self.borrowLTV;
         }
 
-        uint256 currentLTV = self.originalLTV.toUint16();
+        if (block.timestamp >= self.targetTimestamp || self.liquidationLTV >= self.initialLiquidationLTV) {
```

Comment:
i don't think the second check is needed, you always have block.timestamp >= self.targetTimestamp if the liquidationLTV is growing.